Dockerfile.twig

```diff
@@ -49,8 +65,8 @@ RUN apt-get update && apt-get install -y openssl git && apt-get purge -y --auto-
 RUN \
   sed -i "s/^memory_limit = .*/memory_limit = \"\${PHP_CLI_MEMORY_LIMIT}\"/g" "/etc/php/$PHP_VERSION/cli/php.ini"; \
   \
-  sed -i "s/^;?access\.log = .*/access.log = \/proc\/self\/fd\/2/g" "/etc/php/$PHP_VERSION/fpm/pool.d/www.conf"; \
-  sed -i "s/^;?error\.log = .*/error.log = \/proc\/self\/fd\/2/g" "/etc/php/$PHP_VERSION/fpm/php-fpm.conf"; \
+  sed -i "s/^;?access\.log = .*/access.log = {{ php_logger }}/g" "/etc/php/$PHP_VERSION/fpm/pool.d/www.conf"; \
+  sed -i "s/^;?error\.log = .*/error.log = {{ php_logger }}/g" "/etc/php/$PHP_VERSION/fpm/php-fpm.conf"; \
```

Comment:
`/etc/php/$version/cli/php.ini` has the `error_log` config, shall we change it too?